packages/overlays/src/LocalOverlayController.js

```diff
@@ -145,6 +146,9 @@ export class LocalOverlayController {
       this.contentNode.style.zIndex = 1;
       this.invokerNode.setAttribute('aria-expanded', true);
 
+      if (this.isDropdown) {
```

Comment:
I think it should be called something like `this.inheritsInvokerWidth`, any other ideas?

Reply:
Please look at the document you submitted yourself: https://github.com/ing-bank/lion/blob/master/packages/overlays/docs/OverlaySystemScope.md

I would consider this part of the placement config and expect namings to be in line with those already chosen by Popper. From a generic perspective, there is not always an invoker, but a relative positioning element

Reply:
This doc suggests 'bottom-fill', in line with Popper placement. As @jorenbroekema argued, Popper doesn't adjust dimensions, solely positions. In that case we can treat it as some kind of preprocessor and then place it as a 'bottom-middle' (at the current level (direct member of config object)).